### PR TITLE
Add package change scanner

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Script to ensure any psammead change has a changelog, package.json and package-lock.json update

**Code changes:**

- Looks for any diff between current and latest
- Collates all changes and checks if changelog, package.json or package-lock.js are missing
- Throws error with info if some are missing

Runs on jenkins as the last branch check. If needed admin merge can override this. When this happens, latest would be in sync with itself, so no diff would be flagged again.

**Testing notes:**
1) checkout branch
2) Make a random change in a psammead package locally
4) run `npm run changeScanner`
5) You should see 
```
Branch must update CHANGELOG.md in <package>
Branch must update package-lock.json in <package>
Branch must update package.json in <package>
```
-
6) if you change these 3 files
7) run `npm run changeScanner`
8) you should see 
```
All packages have met minmum change requirements 🎉
```

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: scripts/changeScanner/getChanges.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const { exec } = require('shelljs');
+
+const getChanges = () => {
+  const execute = exec(`git diff --name-only latest ./packages`, {
+    silent: true,
+  }).stdout;
+
+  const changedFiles = execute.split('\n').filter(val => val);
```
Comment (jroebu14) [inline scripts/changeScanner/getChanges.js]: if you're checking for truthy things here you could do `.filter(Boolean);` instead

File: scripts/changeScanner/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+});
+
+if (errors.length > 0) {
+  // eslint-disable-next-line no-console
+  errors.forEach(error => console.error(chalk.red(error)));
+  process.exit(1);
+} else {
+  // eslint-disable-next-line no-console
+  console.log(
+    chalk.green('All packages have met minmum change requirements 🎉'),
```
Comment (jroebu14) [inline scripts/changeScanner/index.js]: typo here with `minmum `

**Review by jroebu14 — APPROVED** [review]
nice 👍

File: scripts/changeScanner/getChanges.js (hunk 1 above)
Comment (dr3) [reply scripts/changeScanner/getChanges.js]: Nice, never thought of that :D

**Review by chris-hinds — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
This doesn't need testing

**Pull request merged**